ui/app/mainui/Popups.qml

```diff
@@ -85,8 +85,8 @@ QtObject {
         popup.openPopup(image)
     }
 
-    function openProfilePopup(publicKey: string, parentPopup) {
-        openPopup(profilePopupComponent, {publicKey: publicKey, parentPopup: parentPopup})
+    function openProfilePopup(publicKey: string, parentPopup, cb) {
```

Comment:
Just wondering if this signature change doesn't break opening the profile popup from other places.... :thinking:

Reply:
Good eye. I did test it because I had the same concern. It seemed to work fine in all the places I could think of for the profile. I can paste a video later

Reply:
All good then, I remember we had a similar issue when we moved those popups around :)